src/daft-sql/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[dependencies]
+common-error = {version = "0.2.0-dev0", path = "../common/error"}
+daft-core = {version = "0.2.0-dev0", path = "../daft-core"}
+daft-dsl = {version = "0.2.0-dev0", path = "../daft-dsl"}
+daft-plan = {version = "0.2.0-dev0", path = "../daft-plan"}
```

Comment:
How come the versions have to be pinned like so?

Reply:
oh, i just ran `cargo add -p daft-sql <crate>` and it did this. The version doesn't have to be pinned unless you plan on releasing to crates.io